src/app/fdctl/run/tiles/fd_quic.c

```diff
@@ -410,14 +394,13 @@ quic_stream_notify( fd_quic_stream_t * stream,
 
   if( FD_UNLIKELY( ( slot->conn_id   != conn_id   ) |
                    ( slot->stream_id != stream_id ) ) ) {
-    FD_MCNT_INC( QUIC_TILE, REASSEMBLY_NOTIFY_CLOBBERED, 1UL );
+    FD_MCNT_INC( QUIC_TILE, TXNS_OVERRUN, 1UL );
     return;  /* clobbered */
   }
 
   /* Abort reassembly slot if QUIC stream closes non-gracefully */
 
   if( FD_UNLIKELY( type!=FD_QUIC_STREAM_NOTIFY_END ) ) {
-    FD_MCNT_INC( QUIC_TILE, REASSEMBLY_NOTIFY_ABORTED, 1UL );
```

Comment:
Why removing this case? It's useful to see these in the GUI

Reply:
@mmcgee-jump It's duplicating another metric but is less precise. The GUI code sources these metrics from fd_quic now not the tile.